src/tweenable.rs

```diff
@@ -359,6 +352,29 @@ impl<T> Tween<T> {
         self
     }
 
+    /// TODO
+    pub fn set_repeat_count(&mut self, count: RepeatCount) {
+        self.count = count;
+        self.clock.is_looping = count != RepeatCount::Finite(1);
+    }
```

Comment:
I like these methods from an initialization devx standpoint, but I have no idea what it means to change the repeat values in the middle of a running animation. Not sure what to do about that.

Reply:
Thinking about this a bit more, it's probably fine with the is_looping finikyness removed. It just means you can extend or shorten the animation.